### PR TITLE
CDAP-15550 Netezza db plugin enhacements: all data types support + pr…

### PR DESCRIPTION
**Pull request by vladhlinskiy**
…oper test coverage

### CODE REVIEW

File: netezza-plugin/src/test/java/io/cdap/plugin/netezza/NetezzaPluginTestBase.java [hunk 1]
```diff
@@ -122,92 +128,64 @@ protected static void createTestTables(Connection conn) throws SQLException {
...
-                     "REAL_COL REAL," +
-                     "ST_GEOMETRY_COL ST_GEOMETRY(10)," +
-                     "VARBINARY_COL BINARY VARYING(10)," +
-                     "DECIMAL_COL DECIMAL(" + PRECISION + "," + SCALE + ")," +
-                     "FLOAT_COL FLOAT(6))");
-      stmt.execute("CREATE TABLE MY_DEST_TABLE AS " +
-                     "SELECT * FROM my_table");
-      stmt.execute("CREATE TABLE your_table AS " +
-                     "SELECT * FROM my_table");
+      stmt.execute("create table MY_TABLE (INTEGER_COL INTEGER, BYTEINT_COL BYTEINT, SMALLINT_COL SMALLINT, " +
```
Comment (mlozbin-cybervisiontech) [inline netezza-plugin/src/test/java/io/cdap/plugin/netezza/NetezzaPluginTestBase.java]: what is the reason to reformat this part? Now it is not readable

File: netezza-plugin/src/test/java/io/cdap/plugin/netezza/NetezzaPluginTestBase.java [hunk 2]
```diff
@@ -91,9 +97,9 @@ public static void setupTest() throws Exception {
     setupBatchArtifacts(DATAPIPELINE_ARTIFACT_ID, DataPipelineApp.class);
 
     addPluginArtifact(NamespaceId.DEFAULT.artifact(JDBC_DRIVER_NAME, "1.0.0"),
-                      DATAPIPELINE_ARTIFACT_ID,
-                      NetezzaSource.class, NetezzaSink.class, DBRecord.class, ETLDBOutputFormat.class,
-                      DataDrivenETLDBInputFormat.class, DBRecord.class, NetezzaPostAction.class, NetezzaAction.class);
+      DATAPIPELINE_ARTIFACT_ID,
```
Comment (mlozbin-cybervisiontech) [inline netezza-plugin/src/test/java/io/cdap/plugin/netezza/NetezzaPluginTestBase.java]: Do you use CDAP codestyle settings? Format looks strange.

File: netezza-plugin/src/test/java/io/cdap/plugin/netezza/NetezzaPluginTestBase.java [hunk 3]
```diff
@@ -122,92 +128,64 @@ protected static void createTestTables(Connection conn) throws SQLException {
...
-                                "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?," +
-                                "       ?, ?, ?, ?, ?, ?, ?, ?, ?, ?," +
-                                "       ?, ?, ?, ?, ?)");
-      PreparedStatement pStmt2 =
-        conn.prepareStatement("INSERT INTO your_table " +
-                                "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?," +
-                                "       ?, ?, ?, ?, ?, ?, ?, ?, ?, ?," +
-                                "       ?, ?, ?, ?, ?)")) {
-
+    try (Statement stmt = conn.createStatement()) {
```
Comment (mlozbin-cybervisiontech) [inline netezza-plugin/src/test/java/io/cdap/plugin/netezza/NetezzaPluginTestBase.java]: please try to avoid non-related refactoring

**Review by mlozbin-cybervisiontech — CHANGES_REQUESTED** [review]

**Comment by vladhlinskiy** [comment]
@mlozbin-cybervisiontech all style issues were addressed.

**Review by mlozbin-cybervisiontech — APPROVED** [review]

**Pull request merged**